### DISCUSSION TITLE
mu4e error  gmail config

### DISCUSSION THREAD

**Discussion by ghost** [Lambda-Emacs/lambda-emacs#13] [General]
Error (use-package): mu4e/:config: Wrong number of arguments: mu4e-maildir-shortcuts, 6 Disable showing Disable logging

https://github.com/wereket/lambda-emacs/blob/main/.local/lambda-library/lambda-setup/lem-setup-mail.el#L528

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/89839215/182045275-2ebb8508-5f4b-46e8-8bbf-6d89b8a34b4e.png)

**Reply by mclearc** [reply]
It looks like you need to tweak `mu4e-headers-fields`.  It looks like you need to fix the spacing so it fits on your screen better.

**Comment by ghost** [comment]
the mail setup is not yet eye candy :(

**Reply by mclearc** [reply]
In the end, once you get the spacing fixed,  it should look something like this:
<img width="1558" alt="Screen Shot 2022-07-31 at 22 51 04" src="https://user-images.githubusercontent.com/1502422/182062969-5f4bee0b-e90e-4847-bfdb-45d944428721.png">

**Comment by mclearc** [comment]
> Error (use-package): mu4e/:config: Wrong number of arguments: mu4e-maildir-shortcuts, 6 Disable showing Disable logging
> 
> https://github.com/wereket/lambda-emacs/blob/main/.local/lambda-library/lambda-setup/lem-setup-mail.el#L528

I think you're missing a `setq` and a quote. Here's mine for example:

```emacs-lisp
(setq mu4e-maildir-shortcuts '((:maildir "/Fastmail/Archive"    :key ?a)
                                 (:maildir "/Fastmail/Inbox"      :key ?i)
                                 (:maildir "/Fastmail/Starred"    :key ?w)
                                 (:maildir "/Fastmail/Sent Items" :key ?s)
                                 (:maildir "/UNL/Inbox"           :key ?I)
                                 (:maildir "/UNL/Archive"         :key ?A)
                                 (:maildir "/UNL/Archive1"        :key ?1)
                                 (:maildir "/UNL/Sent"            :key ?S)))
```
